app/main/routes.py

```diff
@@ -193,10 +198,55 @@ def browse():
         filters=browse_parameters,
         browse_type=browse_type,
         results=browse_results,
+        transferring_bodies=transferring_bodies,
+        user_filters=filters,
         num_records_found=num_records_found,
     )
 
 
+def _build_browse_all_filters():
```

Comment:
We could move these to a utils file to keep the routes.py file strictly for routes

Reply:
I moved the changes to a new util file for both filters and sorting logic in a new PR 177